opensearch-jobs.yml

```diff
@@ -113,6 +113,7 @@ instance_groups:
         app_index_settings:
           index.mapping.total_fields.limit: 2000
           index.queries.cache.enabled: "false"
+          index.max_docvalue_fields_search: 200
```

Comment:
Is this currently causing us problems to have the default of 100?

Reply:
No, I ran into it when updating documentation. We talk about updating it to 300 when Trying to retrieve too many docvalue_fields I believe opensearch will be less but i want a value for documentation.

Reply:
We link alot to pull requests on fixes so if we decide to just keep it at default i can close this pull request and put a link to it instead that way

Reply:
Yeah, I'm just wondering if some of the fixes we've made for OpenSearch made the change in this value unnecessary. It might be worth waiting to make this change until or unless we observe that is necessary